loops/loop47.go

```diff
@@ -0,0 +1,27 @@
+//Greatest common Divisor
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	var n1, n2 int =  75, 20
+	remainder := n1 % n2
+	gcd := Gcd(n1, n2, remainder)
+	fmt.Println(gcd)
+}
+
+func Gcd(n1 int, n2 int, remainder int) int {
+	for remainder != 0 {
+		remainder = n1  % n2
+		if remainder == 0 {
```

Comment:
Instead of if else, find retunr Gcd via Euclidean algorithm

Reply:
This code is implementation of euclidean algorithm.

Reply:
Check line #7 https://play.golang.org/p/SmzvkDjYlb
You don't need to use an if else block